Groups: Update docs for posthog-node, add tip and update examples (#2428)
PostHog/posthog.com: contents/docs/user-guides/group-analytics.md

## Known limitations

> - You can't yet use groups some insights or use them in feature flags. This will be added over the coming week(s).

Reviewer: O.o

Author: o_o

> - You can't yet use groups feature flags or in all insights. These features will be added over the coming week(s).

Reviewer: still borked?

```suggestion
- You can't yet use groups with feature flags or insights. These features will be added over the coming week(s).
```
